_docker/lib/export/export_inspector.rb

```diff
@@ -78,7 +110,7 @@ def inspect_export(url_list, export_directory)
       puts "WARNING: Of '#{total_pages}' pages, '#{missing_pages}' are not found in the export."
 
       if should_fail_on_missing_content
-        raise StandardError.new("There are '#{missing_pages}' pages missing from the site export (see warnings for missing content). Failing export process.")
+        raise(StandardError,"There are '#{missing_pages}' pages missing from the site export (see warnings for missing content). Failing export process.")
```

Comment:
I think for consistency we should use the one without parens (pretty sure that's what is being used in other places)